app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -252,8 +254,19 @@ def pod_full_name
     "#{options[:pod_namespace]}/#{options[:pod_name]}"
   end
 
+  def inspector_admin_secret
+    kubeclient = kubernetes_client
+    begin
+      inspector_sa = kubeclient.get_service_account(IMAGE_INSPECTOR_SA, INSPECTOR_NAMESPACE)
+      return inspector_sa.imagePullSecrets[0].name # TODO support multiple imagePullSecrets. This depends on image-inspector support
+    rescue KubeException => e
+      _log.warn("Service Account #{IMAGE_INSPECTOR_SA} does not exist.")
```

Comment:
@enoodle please restrict this only to the specific error (404 I think?). For anything else we fail.